cms/djangoapps/contentstore/rest_api/v2/urls.py

```diff
@@ -1,15 +1,31 @@
 """Contenstore API v2 URLs."""
 
-from django.urls import path
-
-from cms.djangoapps.contentstore.rest_api.v2.views import HomePageCoursesViewV2
+from django.conf import settings
+from django.urls import path, re_path
 
+from cms.djangoapps.contentstore.rest_api.v2.views import home, downstreams
 app_name = "v2"
 
 urlpatterns = [
     path(
         "home/courses",
-        HomePageCoursesViewV2.as_view(),
+        home.HomePageCoursesViewV2.as_view(),
         name="courses",
     ),
+    # TODO: Potential future path.
```

Comment:
At some point, we'd like to be able to give an _upstream_ ID and get the list of all its downstreams. I'm assuming we'll need some kind of new data model to track that before we're able to implement something like that in the future, right?

Reply:
Yep. Indeed, we'll need something like a `DownstreamUpstreamLink` table, which the upstream_sync API writes to and keeps updated.